library/error.c

```diff
@@ -758,6 +762,13 @@ const char * mbedtls_low_level_strerr( int error_code )
             return( "SHA512 - SHA-512 input data was malformed" );
 #endif /* MBEDTLS_SHA512_C */
 
+#if defined(MBEDTLS_SM3_C)
+        case -(MBEDTLS_ERR_SM3_HW_ACCEL_FAILED):
```

Comment:
`MBEDTLS_ERR_SM3_HW_ACCEL_FAILED` is not used in this patch. Could you remove it from this PR?